queue/src/fiskaltrust.Middleware.Localization.QueueES/Models/MiddlewareState.cs

```diff
@@ -0,0 +1,54 @@
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using fiskaltrust.ifPOS.v2;
+
+namespace fiskaltrust.Middleware.Localization.QueueES.Models;
+
+public class MiddlewareState
```

Comment:
These should in the future be defined in the interface or some common project. I already ran into problems during dev when changing it on one side only and not the other.

On the other hand there must never be changes in here that can break anything so we'll need robust regression/snapshot tests for that.